global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.0.100-preview3-010431"
+    "version": "3.0.100-preview4-010737"
```

Comment:
Is this a nightly release? if merged as is, bumping would break the continuous interop test build.

Reply:
We should figure out how to do testing against the latest released preview and against the nightly build at the same time.
Should we have two builds - one against the latest nightly and one against the latest released preview and test against both (and use #ifdefs to make things buildable). Then, once the next preview is out, we can cleanup the #ifdef sites?

Btw, would that nightly have https://github.com/aspnet/AspNetCore/pull/8200?

Reply:
Yes it is a nightly release.

What is the value of testing a build with custom #ifdefs against the official preview? Is it to keep the interop test build passing because it is not possible to install nightly .NET SDKs in that environment?

Reply:
I think the main value is maintain the ability to do a release on nuget.org at any time (there's probably no point in releasing something that won't with the latest official preview) and users still being able to build against the latest preview.

In the interop framework we could pull the never version of SDK if needed, but we'd like to avoid if possible because none of the other gRPC languages does that - all use a fixed environment.

Reply:
Btw, updating the dotnet SDK version in out performance benchmark harness is harder than updating for interop framework - which is yet another motivation to keep things buildable with the stable preview.

Reply:
If we need to make a point release I think it should come from a branch from the 0.1.9 release. I'd rather not try to maintain two versions because we're going to have more and more minor breaking changes from the SDK:
* this rename
* https://github.com/grpc/grpc-dotnet/issues/164
* https://github.com/grpc/grpc-dotnet/issues/144

I don't think we should automatically use the latest nightly each build, but we are going to need to manually update to a newer nightly from time to time. This isn't really any different from manually updating to a newer official preview SDK.

Reply:
Updating should be as simple as running the build/get-dotnet.{ps1/sh} script.